pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+line-length = 88
+target-version = ['py36', 'py37', 'py38']
+include = '\.pyi?$'
+extend-exclude = '''
+/(
+  | external
+  | build
+)/
+'''
```

Comment:
Black's version of a config file where files in the folder external and build are excluded from the formatter.